drivers/sensor/voltage/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Voltage sensor driver
+#
+# Copyright (c) 2023 FTP Technologies
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config VOLTAGE
```

Comment:
I wonder if this would be better called `VOLTAGE_DIVIDER` or `VOLTAGE_SENSOR`. Plain `VOLTAGE` sounds very generic.

Reply:
Done